features/fields_extensie.feature

```diff
@@ -1,126 +1,544 @@
+      | naam        |
+      | inOnderzoek |
+
+    Scenario: persoon heeft geheimhouding en andere gegevens wordt gevraagd met de fields parameter
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                          | waarde    |
+      | burgerservicenummer           | 999993483 |
+      | geheimhoudingPersoonsgegevens | 7         |
+      En de persoon heeft de volgende geboorte gegevens
+      | naam        | waarde |
+      | plaats.code | 0518   |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999993483                       |
+      | fields              | geboorte.plaats                 |
+      Dan bevat de persoon met burgerservicenummer '999993483' de volgende gegevens
+      | naam                          | waarde |
+      | geheimhoudingPersoonsgegevens | true   |
+
+    Scenario: persoon heeft opschorting bijhouding en andere gegevens wordt gevraagd met de fields parameter
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 999992077 |
+      En de persoon heeft de volgende geboorte gegevens
+      | naam        | waarde |
+      | plaats.code | 0518   |
+      En de persoon heeft de volgende overlijden gegevens
+      | naam               | waarde   |
+      | indicatieOverleden | true     |
+      | datum.type         | Datum    |
+      | datum.datum        | 20151001 |
+      | land.code          | 6030     |
+      | plaats.code        | 0344     |
+      En de persoon heeft de volgende opschortingBijhouding gegevens
+      | naam       | waarde     |
+      | reden.code | overlijden |
+      | datum      | 20151001   |
+      En de waardetabel 'Gemeenten' heeft de volgende waarden
+      | code | omschrijving  |
+      | 0518 | 's-Gravenhage |
+      | 0344 | Utrecht
+      En de waardetabel 'Landen' heeft de volgende waarden
+      | code | omschrijving |
+      | 6030 | Nederland    |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 999992077                       |
+      | fields              | geboorte.plaats                 |
+      Dan bevat de persoon met burgerservicenummer '999992077' de volgende geboorte gegevens
+      | naam                | waarde        |
+      | plaats.code         | 0518          |
+      | plaats.omschrijving | 's-Gravenhage |
+      En bevat de persoon met burgerservicenummer '999992077' de volgende opschortingBijhouding gegevens
+      | naam              | waarde     |
+      | reden.code        | overlijden |
+      | reden.datum.datum | 2015-10-01 |
```

Comment:
dit veld bestaat niet, reden zou eruit moeten.

ik vind dat we alleen opschortingBijhouding.reden ongevraagd moeten leveren, zie opmerking daarover bovenaan

Reply:
Datum  opschortingBijhouding is er uitgehaald